src/utils/columns.js

```diff
@@ -224,3 +245,47 @@ export default function getColumns(props) {
     },
   ]
 }
+
+export const getPositionsTickersColumns = (props) => {
+  const { filteredData } = props
+
+  return [
+    {
+      id: 'pair',
+      name: 'tickers.column.pair',
+      width: COLUMN_WIDTHS.PAIR,
+      renderer: (rowIndex) => {
+        const { pair } = filteredData[rowIndex]
+        return (
+          <Cell tooltip={pair}>
+            {pair}
+          </Cell>
+        )
+      },
+      copyText: rowIndex => filteredData[rowIndex].pair,
```

Comment:
does `filteredData` guarantee to have `rowIndex` ? probably should guard with `_get` if not to avoid runtime error

if it is guarantee then 🙆‍♂

Reply:
@jarindr yes, should be guaranteed by the library, used only in provided functions like:
https://github.com/bitfinexcom/bfx-report-ui/blob/master/src/ui/DataTable.js#L18